hack/svc/multi-stream.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import shutil
+import time
+import re
+import getopt
+import sys
+from subprocess import Popen
+
+
+class strims_ffmpeg:
+    def __init__(self, hostAddress="0.0.0.0") -> None:
+        self.hostAddress = hostAddress

```

Comment:
```suggestion
        self.host_address = hostAddress
```

prefer snake case for variable names (throughout the file)

Reply:
all variables should be named with snake_case